bedrock/firefox/templates/firefox/australis/fx36/tour-base.html

```diff
@@ -8,6 +8,23 @@
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {% if l10n_has_tag('meta_update_201503') %}
```

Comment:
Since we're using these 2 strings in 3 files, it makes more sense to have them in a shared file (and we can't have tags on those).

So, let's remove the l10n_has_tag conditional on these pages
```
bedrock/firefox/templates/firefox/australis/fx36/tour-base.html
bedrock/firefox/templates/firefox/australis/tour-base.html
bedrock/firefox/templates/firefox/new.html
```
I'll move the strings currently in firefox/new.lang into main.lang